TrafficCapture/nettyWireLogging/src/main/java/org/opensearch/migrations/trafficcapture/netty/LoggingHttpRequestHandler.java

```diff
@@ -126,11 +128,10 @@ public void channelInactive(ChannelHandlerContext ctx) throws Exception {
 
     protected void channelFinishedReadingAnHttpMessage(ChannelHandlerContext ctx, Object msg, HttpRequest httpRequest) throws Exception {
         super.channelRead(ctx, msg);
-        metricsLogger.atSuccess()
-                .addKeyValue("channelId", ctx.channel().id().asLongText())
-                .addKeyValue("httpMethod", httpRequest.method().toString())
-                .addKeyValue("httpEndpoint", httpRequest.uri())
-                .setMessage("Full request received").log();
+        metricsLogger.atSuccess(MetricsEvent.RECEIVED_FULL_HTTP_REQUEST)
+                .setAttribute(MetricsAttributeKey.CHANNEL_ID, ctx.channel().id().asLongText())
```

Comment:
At some point in the future, it will be nice to have typesafe helpers to do these transformations in one spot.